Bugzilla/WebService/Bug.pm

```diff
@@ -632,7 +632,7 @@ sub search {
             return $data;
         }
         else {
-            return { bug_count => $data };
+            return { bug_count => $self->type('int', $data) };
```

Comment:
To be double sure that we do the right thing, let's change it to 
return { bug_count => $self->type('int', scalar @$data) };

Reply:
Fixed!